demos/wasmcloud/greet/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "greet"
+version = "0.6.0"
+authors = ["Matthew Fisher <matt.fisher@microsoft.com"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+wasmcloud-provider-core = "0.1"
```

Comment:
You shouldn't need the provider core dep for an actor, perhaps this is leftover from an old wascc-codec dependency?

Reply:
More than likely that is correct